packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -671,6 +671,14 @@ class _FocusState extends State<Focus> {
     Widget child = widget.child;
     if (widget.includeSemantics) {
       child = Semantics(
+        // Automatically request the focus for a focusable widget when it
+        // receives accessibility focus. Nothing is needed for losing the
+        // accessibility focus because if focus is lost, that means another node
+        // will gain focus and take focus from this widget.
```

Comment:
The comment still refers to "accessibility focus". Since we switched from `didGainAccessibilityFocus` to `focus`, it should say just "focus", or to be more precise "input focus request from semantics".